src/obb_rules/game.cljc

```diff
@@ -28,18 +30,23 @@
    (assoc game :state new-state)))
 
 (defn create
-  "Creates a game for a given stash"
-  [stash]
-  (let [board (board/create-board)
-        board1 (board/set-stash board :p1 stash)
-        board2 (board/set-stash board1 :p2 stash)]
-    (assoc board2 :state :deploy)))
+  "Creates a game for the given stashes.
+   stashes is an associative collection in which the keys correspond to the
+   players and the values to the corresponding stash."
+  [stashes]
+  (-> (reduce-kv (fn [board player stash] (board/set-stash board player stash))
```

Comment:
Much better style I must say! At some point the function was slightly more complicated and when I simplified it, I probably missed that I didn't need that boilerplate anymore :+1: